docs/integrations/vulnerability-scanners/trivy.md

```diff
@@ -92,6 +92,8 @@ EOF
 | `trivy.serverURL`                 | N/A                                | The endpoint URL of the Trivy server. Required in `ClientServer` mode. |
 | `trivy.serverTokenHeader`         | `Trivy-Token`                      | The name of the HTTP header to send the authentication token to Trivy server. Only application in `ClientServer` mode when `trivy.serverToken` is specified. |
 | `trivy.insecureRegistry.<id>`     | N/A                                | The registry to which insecure connections are allowed. There can be multiple registries with different registry `<id>`. |
+| `trivy.mirrors.registry.<id>`     | N/A                                | A registry for which a mirror should be used to get an image. For the mirror url `trivy.mirrors.mirror.<id>` with the matching `<id>`. For each entry there must be a corresponding `trivy.mirrors.mirror.<id>` entry. There can be multiple registries with different registry `<id>`. |
+| `trivy.mirrors.mirror.<id>`       | N/A                                | The mirror for the registry with `<id>`. |
```

Comment:
I am not quite sure how to frame this best. I updated the docs, since now there is only one option and I added a example to make it more clear.

But I am open to other suggestions.